fetch/src/main/java/com/indix/gocd/s3fetch/FetchExecutor.java

```diff
@@ -72,4 +72,7 @@ public AmazonS3Client s3Client(FetchConfig config) {
         return client;
     }
 
+    public FetchConfig getFetchConfig(TaskConfig config, TaskExecutionContext context) {
```

Comment:
Why do we have getFetchConfig which doesn't seem to be doing anything as a separate function?